lib/config/genconfig/awshaprovisionconfig.go

```diff
@@ -29,8 +29,8 @@ const (
 	AWS_ACCESS_KEY_SECRET_REGEX        = "^[A-Za-z0-9/+=]{40}$"
 	ACCESS_KEY_ID_REGEX                = "^[A-Za-z0-9/+=]+$"
 	ACCESS_KEY_SECRET_REGEX            = "^[A-Za-z0-9/+=]+$"
-	ENDPOINT_URL                       = "^((http|https)://)[-a-zA-Z0-9@:%._\\+~#?&//=]{2,256}\\.[a-z]{2,6}\\b([-a-zA-Z0-9@:%._\\+~#?&//=]*)$"
-	ENDPOINT_URL_SAMPLE                = "http://bucket-endpoint.com or https://bucket-endpoint.com"
+	ENDPOINT_URL                       = "^(https?://(?:[a-zA-Z0-9-]+\\.)+[a-zA-Z]{2,}|https?://(?:[0-9]{1,3}\\.){3}[0-9]{1,3}:[0-9]{1,5})$"
```

Comment:
Modify the regex to support DNS with port
Eg: https://mydomin.com:5432

Reply:
Modified